main/templates/users/list.html

```diff
@@ -1,5 +1,41 @@
 {% extends "layouts/master_sidebar.html" %}
 {% load static %}
+{% block botonPrimario %}
+{% if 'asem/list' in request.path %}
```

Comment:
I would change these ifs to {% if object_name = '<x>' %}, being <x> the object_name property put into context in person/views.py because now, unlike in master_sidebar, you have access to this specific data that doesn't depend on the url and may be more reliable in the long run. For example, for the asem/create if, it would be {% if object_name = 'usuario' %}